lua/nvim-tree/explorer/init.lua

```diff
@@ -13,6 +13,7 @@ Explorer.__index = Explorer
 
 function Explorer.new(cwd)
   cwd = cwd or uv.cwd()
+  vim.cmd('cd '..cwd)
```

Comment:
I'm not sure nvim-tree should manage your base directory, also this would loop when `:cd dir` and also crash when doing `nvim dir`